python/mainwindow.py

```diff
@@ -1216,7 +1217,8 @@ def OnInit(self):
     def _executableFound(self, executable):
         devnull = open('/dev/null', 'wb')
         try:
-            returncode = subprocess.call(["which", executable], stdout=devnull)
+            executable_path = shutil.which(executable)
+            returncode = subprocess.call(executable_path, stdout=devnull)
```

Comment:
I am sorry, but this is incorrect: `returncode` gets the exit code from calling e.g. `/usr/bin/nc` assigned, which is in my case `2` (when using `ncat` from nmap, the default at various Linux distributions, such as Fedora). This return code says nothing whether the desired executable exists or whether it's usable.

Reply:
You are correct. We should use the expression `shutil.which(executalbe) is None` instead

Reply:
(or is not None)

Reply:
I've proposed a fix with #74, just let me know if you prefer another logic or code style or so.